plugin/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,17 @@
+name: "Temporal"
+description: "Temporal is a developer-first, open source platform that ensures the successful execution of services and applications."
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  short-name: "temporal"
+  keywords:
+    - "workflow"
+    - "orchestration"
+    - "cron"
+    - "state-machine"
+  categories:
+    - "integration"
+  config:
+    - "quarkus.temporal."
+  status: "preview"
+  guide: "https://quarkiverse.github.io/quarkiverse-docs/quarkus-temporal/dev/index.html"
+  icon-url: "https://raw.githubusercontent.com/quarkiverse/quarkus-temporal/main/docs/modules/ROOT/assets/images/temporal_logo.svg"
```

Comment:
Does this point to something yet ?

Reply:
the icon is there the only thing it doesn't point to is the GUIDE but it will eventually be at that URL when we are all done.